generator-mediator-js/package.json

```diff
@@ -28,11 +29,13 @@
     "Interoperability"
   ],
   "dependencies": {
-    "yeoman-generator": "^0.18.0",
     "chalk": "^0.5.0",
-    "yosay": "^0.3.0",
     "mocha": "*",
     "node-uuid": "^1.4.2",
-    "yo": ">=1.0.0"
+    "yeoman-assert": "^2.2.1",
+    "yeoman-generator": "^0.23.0",
+    "yeoman-test": "^1.4.0",
+    "yo": ">=1.0.0",
+    "yosay": "^0.3.0"
   }
 }
```

Comment:
Can you make sure this file follows the guidelines specified [here](http://yeoman.io/generators/) under 'How to add your generator' so that this generator will show up there.